docs/user-manual/civiform-admin-guide/manage-questions.md

```diff
@@ -86,7 +86,7 @@ You can customize your program to include multiple different question types. The
 | ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Address | An Applicant’s address. For example, residential, work, mailing, school, etc. |
 | Currency | Currency questions are formatted with a decimal point (.) as a separator between the dollars and cents. Useful for asking income and debt-related questions (e.g. wages, benefits received, loan amounts, etc). |
-| Checkbox | Useful when Applicants need to check multiple items to answer the question fully. <strong>Tip</strong>: If you want Applicants to select only one option from a group of options, use a Dropdown or Radio Button question instead. |
+| Checkbox | Useful when Applicants need to check multiple items to answer the question fully.<br><strong>Tip</strong>: If you want Applicants to select only one option from a group of options, use a Dropdown or Radio Button question instead.<br><strong>Tip</strong>: If a checkbox question is required, then Applicants <strong>must</strong> select at least one answer. For checkbox questions where an Applicant could have no answer, consider including a "None of these" option or making the checkbox question optional. |
```

Comment:
```suggestion
| Checkbox | Useful when Applicants need to check multiple items to answer the question fully.<br><strong>Tip</strong>: If you want Applicants to select only one option from a group of options, use a Dropdown or Radio Button question instead.<br><strong>Tip</strong>: If a Checkbox question is required, then Applicants <strong>must</strong> select at least one answer. For checkbox questions where an Applicant could have no answer, consider including a "None of these" option or making the checkbox question optional. |
```

Reply:
I don't know if it will work in the table, but GitBook provides hint styles for paragraphs:
https://docs.gitbook.com/content-editor/blocks/hint